### DISCUSSION TITLE
Register this app as an authorised app source

### DISCUSSION THREAD

**Discussion by robertosw** [Droid-ify/client#605] [General]
I just upgraded to Android 14 on a Samsung Device. So I am not sure if the following is something that only applies to Samsung devices.
In Settings > Security and Privacy > Auto Blocker
 is a new setting which blocks installing apps from unauthorised apps, enables checking running apps on suspicious activity, and blocks harmful commands over usb cable.
This seems to be quite a nice security feature, but it only allows the Google PlayStore and Samsung Store to install apps. 
I'd be nice to have Droidify / F-Droid authorised, but I dont know how this can be done.

**Comment by robertosw** [comment]
This is the setting page
![Screenshot_20231213_103952_Auto Blocker](https://github.com/Droid-ify/client/assets/47303535/fdada591-96a8-40e4-b871-036bf0c17553)

**Reply by Iamlooker** [reply]
Can you click on "Learn More" under it and send me the screenshot, or the link of its a webpage.

**Reply by robertosw** [reply]
![Screenshot_20231213_133221_Settings](https://github.com/Droid-ify/client/assets/47303535/f27a2e4d-f9aa-48fe-802b-95a49592a2e8)

**Reply by Iamlooker** [reply]
Is this where it goes when you click on learn more under authorised apps? If yes then thats very weird

**Reply by robertosw** [reply]
Yeah, it just opens this settings page 😅

**Reply by Iamlooker** [reply]
I will look into this in somedays 🤔

**Reply by robertosw** [reply]
Thank you 👍

**Comment by realeatham** [comment]
I doubt you can get Samsung to say it's authorized, just turn auto blocker off.
